scripts/integration-test

```diff
@@ -9,123 +9,50 @@ set -e
 export SCRIPTS_ROOT
 source "${SCRIPTS_ROOT:?}"/util.sh
 
-SSH_TUNNEL_SOCKET=/tmp/prose-tests
-
-close-ssh-tunnel() {
-	ssh -S "${SSH_TUNNEL_SOCKET:?}" -O exit whatever 2>/dev/null || :
-}
-on-exit() {
-	close-ssh-tunnel
-}
-trap 'on-exit' EXIT
```

Comment:
I removed all of the logic related to running integration tests on a remote RPi which results in cleaner code.